go.mod

```diff
@@ -16,3 +16,5 @@ retract (
 	v0.65.0
 	v0.37.0 // Contains dependencies on v0.36.0 components, which should have been updated to v0.37.0.
 )
+
+replace github.com/open-telemetry/opentelemetry-collector => ../opentelemetry-collector
```

Comment:
This only works locally. Tests need to be fixed in http://github.com/open-telemetry/opentelemetry-collector-contrib/pull/40452